pkg/cert/cert.go

```diff
@@ -36,14 +36,21 @@ const (
 //+kubebuilder:rbac:groups="admissionregistration.k8s.io",resources=validatingwebhookconfigurations,verbs=get;list;watch;update
 
 // CertsManager creates certs for webhooks.
-func CertsManager(mgr ctrl.Manager, namespace string, setupFinish chan struct{}) error {
+func CertsManager(mgr ctrl.Manager, namespace string, configServiceName string, configSecretName string, setupFinish chan struct{}) error {
+	if configServiceName == "" {
+		configServiceName = serviceName
+	}
+	if configSecretName == "" {
+		configSecretName = secretName
+	}
+
```

Comment:
can we remove this and the definitions above (lines 25 and 26), just so we don't create confusion and have defaults in two places.

Reply:
Makes sense. Done.